test/Autofac.Extensions.DependencyInjection.Test/HostBuilderExtensionsTests.cs

```diff
@@ -1,5 +1,7 @@
-﻿using Microsoft.Extensions.DependencyInjection;
+﻿using Autofac.Core.Lifetime;
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Microsoft.VisualBasic;
```

Comment:
Is this supposed to be here?

Reply:
Wonder how I got that there! Removed it.